cppman/main.py

```diff
@@ -113,33 +123,47 @@ def rebuild_index(self):
                                        '(id INTEGER NOT NULL PRIMARY KEY, title VARCHAR(255) NOT NULL UNIQUE, url VARCHAR(255) NOT NULL UNIQUE)' % table)
                 self.db_cursor.execute('CREATE TABLE "%s_keywords" '
                                        '(id INTEGER NOT NULL, keyword VARCHAR(255), FOREIGN KEY(id) REFERENCES "%s"(id))' % (table, table))
-                # crawl and insert all entries
```

Comment:
can you also fix the above two queries as well?

Reply:
yes of course, I fixed a few more lines.